src/connection.js

```diff
@@ -1245,6 +1286,7 @@ Connection.prototype.STATE = {
     name: 'SentClientRequest',
     exit: function() {
       this.clearRequestTimer();
+      this.tokenStreamParser.resume();
```

Comment:
> As long as Connection.execSql() has not yet been called, the request does not know the connection.

That's true. You could call this.request.resume() to keep the paused state in a consistent manner between Request class and Connection class. If that were to be the only concern...

> So the paused state of the Request objects are independent of the paused state of the token parser transform

This boils down to Request classes decides if the Request is paused state. And Connection class decides what that actually means. Sounds like a reasonable separation to me.